mailet/standard/src/main/java/org/apache/james/transport/mailets/SerialiseToHTTP.java

```diff
@@ -118,16 +118,14 @@ public void init() throws MessagingException {
     @Override
     public void service(Mail mail) {
         try {
-            MimeMessage message = mail.getMessage();
-            String serialisedMessage = getSerialisedMessage(message);
-            NameValuePair[] nameValuePairs = getNameValuePairs(serialisedMessage);
-            String result = httpPost(nameValuePairs);
+            String result = httpPost(
+                    getNameValuePairs(MimeMessageUtil.asString(mail.getMessage())));
             if (passThrough) {
                 addHeader(mail, (result == null || result.length() == 0), result);
             } else {
                 mail.setState(Mail.GHOST);
             }
-        } catch (MessagingException | IOException me) {
+        } catch (Exception me) {
```

Comment:
This exception is too broad in my opinion

Reply:
This code snippet it's outdate, but i agree with you, in the new code, i try to isolate the exception only for the call to `MimeMessageUtil.asString(message);` this method throws Exception, however, if you think it's not correct, i can write a new method in the class, and only throw Messaging and IO exceptions.